src/translations/en.global.json

```diff
@@ -206,13 +206,48 @@
         }
     },
 
+    "dashboardActivity": {
+        "title": {
+            "CreateAccount": "New account created",
+            "DeleteAccount": "Account deleted",
+            "DeployContract": "Contract deployed",
+            "FunctionCall": "Method called",
+            "Transfer": {
+                "transferred": "Sent NEAR",
+                "received": "Received NEAR"
+            },
+            "Stake": "Staked",
+            "AddKey": {
+                "forContract": "Access Key added",
+                "forReceiver": "Access Key added"
+            },
+            "DeleteKey": "Key deleted"
+        },
+        "message": {
+            "CreateAccount": "<span>${receiverId}</span>",
+            "DeleteAccount": "<span>${receiverId}<span>",
+            "DeployContract": "<span>${receiverId}</span>",
+            "FunctionCall": "<span>${methodName}</span> in contract <span>${receiverId}</span>",
+            "Transfer": {
+                "transferred": "to <span>${receiverId}</span>",
+                "received": "from <span>${signerId}</span>"
+            },
+            "Stake": "<span>${stake}</span>",
+            "AddKey": {
+                "forContract": "for <span>${permissionReceiverId}</span>",
+                "forReceiver": "for <span>${receiverId}</span>"
+            },
+            "DeleteKey": ""
```

Comment:
why is DeleteKey translation empty? Is it by design?

Reply:
@vgrichina it was like this in the previous version - do you have a proposition on what to add there?